tests/fastify-controller.js

```diff
@@ -15,17 +16,17 @@ test('lifecycle', async (t) => {
     { host: '0.0.0.0' },
   ]
 
-  for (const opts of startOptsFixtures) {
-    await fastifyController.start(opts)
-    await fastifyController.start(opts)
-    await fastifyController.stop()
-    await fastifyController.stop()
+  assert.doesNotReject(async () => {
```

Comment:
think the usage of assert can be skipped entirely in this case

> Using assert.doesNotReject() is actually not useful because there is little benefit in catching a rejection and then rejecting it again. Instead, consider adding a comment next to the specific code path that should not reject and keep error messages as expressive as possible.

https://nodejs.org/docs/latest-v20.x/api/assert.html#assertdoesnotrejectasyncfn-error-message

Reply:
Good point. Changed in 49397fda0d020974b7fb276de2f117efea74cdd9.